packages/core/src/configuration/ConfigurationHelper.ts

```diff
@@ -17,10 +17,18 @@ export class ConfigurationHelper {
 
   /**
    * Get a service configuration object, provided an existing id is given
-   * @param id string - The id of the service
+   * @param type - The type of the service
    */
-  public getConfig(id: string): ServiceConfiguration | undefined {
-    return find(this.configurations, service => service.id.toLowerCase() === id.toLowerCase());
+  public getConfig(type: string): ServiceConfiguration | undefined {
+    return find(this.configurations, service => service.type.toLowerCase() === type.toLowerCase());
+  }
+
+  /**
+   * Get a service configuration object, provided an existing id is given
+   * @param id - unique id of the service
+   */
+  public getConfigById(id: string): ServiceConfiguration | undefined {
+    return find(this.configurations, service => service.type.toLowerCase() === id.toLowerCase());
```

Comment:
`service.type` should not be `service.id`?

Reply:
`id` is generated by OpenShift so that it is not fixes, type on the other hand will be always `metrics`, `auth`, etc.

Reply:
but this method is called `getConfigById` and it should perform the look up by id? Look up by `type` is done in the method above.